referral requests/README.md

```diff
@@ -0,0 +1,173 @@
+| `RequestStatement / priorityCode [@code='394848005'] [@displayName='Normal'] [@codeSystem='2.16.840.1.113883.2.1.3.2.4.15']` | `ReferralRequest.priority` when `priority` = `ROUTINE`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| `RequestStatement / priorityCode [@code='394849002'] [@displayName='High'] [@codeSystem='2.16.840.1.113883.2.1.3.2.4.15']`   | `ReferralRequest.priority` when `priority` = `URGENT`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| `RequestStatement / responsibleParty [@typeCode="RESP"] / agentRef [@classCode="AGNT"] / id [@root]`                         | `ReferralRequest.recipient.agentref.id`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| reference to [Organisation](../organisations/README.md) `RequestStatement / Participant [@typeCode='AGNT'] / agentRef `      | `ReferralRequest.requester.agent.reference` & `ReferralRequest.onBehalfOf`  when agent is `practictioner` and `onBehalfOf' is reference to `Organization`                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
do you think it would be better to have the link in the next colum? or after the path? Just to make it clearer